pallets/runtime/tests/src/settlement_test.rs

```diff
@@ -3060,3 +3076,433 @@ fn multiple_custodian_settlement() {
             );
         });
 }
+
+// ----------------------------------------- Confidential transfer tests -----------------------------------
+
+/// Creates a mercat account and returns its secret part (to be stored in the wallet) and
+/// the account creation proofs (to be submitted to the chain).
+pub fn gen_account(
+    tx_id: u32,
+    mut rng: &mut StdRng,
+    token_name: &[u8],
+    valid_asset_ids: Vec<AssetId>,
+) -> (SecAccount, PubAccountTx) {
+    // These are the encryptions keys used by MERCAT and are different from the signing keys
+    // that Polymesh uses.
+    let elg_secret = ElgamalSecretKey::new(Scalar::random(&mut rng));
+    let elg_pub = elg_secret.get_public_key();
+    let enc_keys = EncryptionKeys {
+        pblc: elg_pub.into(),
```

Comment:
To the wishlist: why not add vowels? `public`

Reply:
https://polymath.atlassian.net/browse/CRYP-173